gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.daemon=true
 org.gradle.parallel=true
 group=com.readytalk.gradle
-version=0.3.0
+version=0.4.0
```

Comment:
I believe the new features (the versioning plugins) represent a minor version update.